content/docs/insights/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Pulumi Insights
+linktitle: Pulumi Insights
+docs_home: true
+notitle: true
+menu:
+  insights:
+    identifier: insights-home
+    weight: 1
+expanded_menu_ids:
+    - insights-get-started
+    - insights-concepts
+meta_desc: Pulumi Insights provides intelligent infrastructure management with AI-powered, asset management, and compliance remediation across all your cloud resources.
+meta_image: /images/docs/meta-images/insights-meta.png
+h1: Pulumi Insights Documentation
+description: |
+  Pulumi Insights offers an intelligent platform for discovering, understanding, and managing cloud infrastructure with AI-driven analysis, compliance enforcement, and more!
+
+link_buttons:
+  primary:
+    label: Get Started
+    link: /docs/insights/get-started/
+  secondary:
+    label: Create an Account
+    link: https://app.pulumi.com/signup
+
+sections:
+- type: flat
+  heading: Overview
+  description_md: |
+    Pulumi Insights enables you to manage and optimize cloud infrastructure with AI-powered tools for discovery, compliance, and cost efficiency. It brings visibility across all resources, including those not provisioned by Pulumi, such as AWS CloudFormation, Microsoft ARM, and Terraform-managed infrastructure.
```

Comment:
> Terraform-managed infrastructure
change to just 
> terraform

Reply:
thx; adjusted the copy in this latest commit